Failed to publish some packages due to intermittent "npm publish" 403 error

After merging the previous PR #355 I attempted to publish new versions of all affected packages.  The `npm publish` step worked for the first two (build and compile packages) but failed on the third (runtime):

```
npm notice 📦  @sdeverywhere/runtime@0.2.1
npm notice === Tarball Contents === 
npm notice 20.9kB ./dist/index.cjs  
npm notice 1.1kB  LICENSE           
npm notice 5.6kB  README.md         
npm notice 20.9kB dist/index.cjs    
npm notice 45.7kB dist/index.cjs.map
npm notice 16.1kB dist/index.d.cts  
npm notice 16.1kB dist/index.d.ts   
npm notice 19.3kB dist/index.js     
npm notice 45.4kB dist/index.js.map 
npm notice 1.2kB  package.json      
npm notice === Tarball Details === 
npm notice name:          @sdeverywhere/runtime                   
npm notice version:       0.2.1                                   
npm notice filename:      @sdeverywhere/runtime-0.2.1.tgz         
npm notice package size:  42.3 kB                                 
npm notice unpacked size: 192.4 kB                                
npm notice shasum:        b98ed509b8b99e44955f97408747ab2d9c57610d
npm notice integrity:     sha512-xp/aCCcXY9Txa[...]txsoAszISs9vg==
npm notice total files:   10                                      
npm notice 
npm notice Publishing to https://registry.npmjs.org/
npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/@sdeverywhere%2fruntime - You cannot publish over the previously published versions: 0.2.1.
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy, or
npm ERR! 403 on a server you do not have access to.
```

This despite the fact that I didn't actually publish that version already.  Others have run into similar issues:
https://github.com/npm/cli/issues/5058
https://github.com/govuk-react/govuk-react/discussions/1146
https://twitter.com/jon_neal/status/1141717238577356800

The only way to deal with this is to do a version bump and publish for each of the affected packages.

It's possible this was caused in part by the fact that we were using `pnpm run` to do the publish step, which may potentially run some commands in parallel.  I've since changed that to use `--workspace-concurrency=1` to ensure that we publish in serial fashion, but not sure yet if that was the root cause.

--- a/packages/plugin-config/CHANGELOG.md
+++ b/packages/plugin-config/CHANGELOG.md
@@ -2,7 +2,6 @@
 
 ## [0.2.1](https://github.com/climateinteractive/SDEverywhere/compare/plugin-config-v0.2.0...plugin-config-v0.2.1) (2023-05-03)
 
-
 ### Bug Fixes
 
 * add inputId property to InputSpec interface ([#321](https://github.com/climateinteractive/SDEverywhere/issues/321)) ([f461433](https://github.com/climateinteractive/SDEverywhere/commit/f461433df9ae013e73a76a1103c9cb8a5d22ab52)), closes [#320](https://github.com/climateinteractive/SDEverywhere/issues/320)

--- a/packages/plugin-vite/CHANGELOG.md
+++ b/packages/plugin-vite/CHANGELOG.md
@@ -20,7 +20,6 @@ of whether your project is configured to use Vite 3 or Vite 4.
 
 ## [0.1.4](https://github.com/climateinteractive/SDEverywhere/compare/plugin-vite-v0.1.3...plugin-vite-v0.1.4) (2022-10-25)
 
-
 ### Bug Fixes
 
 * make vite a peer dependency for plugin-vite ([#269](https://github.com/climateinteractive/SDEverywhere/issues/269)) ([eaa02fc](https://github.com/climateinteractive/SDEverywhere/commit/eaa02fcb160735ea591f6074cecb662d1b24289c)), closes [#268](https://github.com/climateinteractive/SDEverywhere/issues/268)

--- a/packages/runtime/CHANGELOG.md
+++ b/packages/runtime/CHANGELOG.md
@@ -2,7 +2,6 @@
 
 ## [0.2.1](https://github.com/climateinteractive/SDEverywhere/compare/runtime-v0.2.0...runtime-v0.2.1) (2023-09-28)
 
-
 ### Features
 
 * add support for capturing data for any variable at runtime ([#355](https://github.com/climateinteractive/SDEverywhere/issues/355)) ([5d12836](https://github.com/climateinteractive/SDEverywhere/commit/5d1283657ba99f6c7f8e30f8053f1906ac872af3)), closes [#105](https://github.com/climateinteractive/SDEverywhere/issues/105)

